pants.ini

```diff
@@ -62,7 +62,6 @@ backend_packages: +[
     "pants.contrib.googlejavaformat",
     "pants.contrib.jax_ws",
     "pants.contrib.scalajs",
-    "pants.contrib.mypy",
```

Comment:
Why make this change?

Reply:
This is needed because the existing
```
./pants --exclude-target-regexp='testprojects/.*' --changed-parent="${MERGE_BASE}" lint
```
will fail on c++ related code on resolving different markdown versions, and we don't have a good way to avoid it. You can sanity check by
```
./pants --pants-backends=pants.contrib.mypy --exclude-target-regexp='testprojects/.*' lint ::
```

Internally we can still enable it by default because we don't lint/test a bunch unrelated c++ targets in the same context.